### PR TITLE
XDR IOCs - custom severity

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-16386

### CODE REVIEW

**Comment by Ni-Knight** [comment]
@dorschw - severity field is masked :)

**Review by DeanArbel — APPROVED** [review]

**Comment by Ni-Knight** [comment]
Tests look ok on my side as well.

File: Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py [hunk 1]
```diff
@@ -471,15 +503,25 @@ def get_sync_file():
...
+
+
+def main():  # pragma: no cover
     params = demisto.params()
     Client.severity = params.get('severity', '').upper()
-    Client.query = params.get('query', Client.query)
-    Client.tag = params.get('feedTags', params.get('tag', Client.tag))
     Client.tlp_color = params.get('tlp_color')
+
+    # In this integration, arguments are set in the *class level*, the defaults are in the class definition.
```
Comment (BEAdi) [inline Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py:515]:
```suggestion
    # In this integration, parameters are set in the *class level*, the defaults are in the class definition.
```
Right?

Comment (dorschw) [reply Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py:515]: absolutely

**Pull request merged**